### PR TITLE
modified usage of words from argument to parameter based on the title

### LINKED ISSUE #40124: argument vs parameter unclear usage of words
**Describe your problem and how to reproduce it:**

The usage of the word "argument" ist unclear to me. The title of this challenge uses the word "parameter".

Then the instructions state:
"If an arrow function has a single argument, the parentheses enclosing the argument may be omitted."

But the following snippet cited from the instructions shows the **parameter** "item" without parentheses:
```
// the same function, without the argument [sic] parentheses
const doubler = item => item * 2;
```
The functions are never invoked with an **argument** in the examples.

As I understand it the sentence should say: "If an arrow function has a single **parameter**, the parentheses enclosing the **parameter** may be omitted."

**Add a Link to the page with the problem:**
https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/es6/write-arrow-functions-with-parameters

or,

https://github.com/freeCodeCamp/freeCodeCamp/blob/master/curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/write-arrow-functions-with-parameters.md

**Tell us about your browser and operating system:**
* Browser Name: 
* Browser Version: 
* Operating System: 

**If possible, add a screenshot here (you can drag and drop, png, jpg, gif, etc. in this box):**

### PR DESCRIPTION
**Pull request by SPineapple**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #40124

### CODE REVIEW

**Review by naomi-lgbt — APPROVED** [review]
LGTM~! 🥳 Congratulations on making your first pull request!

As a note, our workflow requires at least one additional approval before this can be merged.

**Review by majestic-owl448 — APPROVED** [review]
LGTM! and I am going to merge this, so congratulations on your first contribution!

**Pull request merged**
